### DISCUSSION TITLE
How long till "Number of pending channels exceed maximum" clears?

### DISCUSSION THREAD

**Discussion by DerEwige** [lightningnetwork/lnd#6907] [Troubleshooting] [answered]
I tried to open a channel to a lnd node.

The operation timed out on my side, but on the lnd side it apparently worked:
When I try to open a channel to the same lnd node now I get this erorr:

`peer sent error: ascii='Number of pending channels exceed maximum`' 

So is the lnd node is waiting for me to publish the channel funding transaction?
But that will never happen as my node thinks the ldn node did not replay.

So how long will lnd keep this pending channel in it's memory before it forgets about it?

**Comment by guggero** [comment, marked as answer]
Channels that never finished the negotiation should be cleared after 10 minutes (or a restart).

**Reply by DerEwige** [reply]
@guggero 
So, if the error still persists? 
Does it mean it was not my channel that ist still pending, but possible just a global issue as the node is very big and gets many requests for new channels?

**Reply by guggero** [reply]
Possible, yes. Or they set their maximum number of pending channels to 0 (meaning they don't accept inbound channels).
